tests/unit/test_hyper_transformer.py

```diff
@@ -14,6 +14,52 @@
 
 class TestHyperTransformer(TestCase):
 
+    def test__add_field_to_set_string(self):
```

Comment:
Need to add these tests to get 100% coverage (I guess only the multi-column tests used the tuple case of this method, so deleting the tests decreased the coverage.)